### PR TITLE
CIFAR-10 || Custom ResNet-9 || PyTorch C++ JIT

### PR DESCRIPTION
**Pull request by LaurentMazare**
Running inference on CIFAR-10 using a trimmed-down version of David Page ResNet-9 architecture. This uses PyTorch JIT scripts via the C++ api to execute the trained model. Each of the 10000 images from the test set is processed independently and only after that the previous image has been processed entirely.
The resulting timing on a P100 is ~0.828 milliseconds per image.

### CODE REVIEW

File: CIFAR10/inference/laurentmazare_resnet9_1p100_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": "v1.0",
+    "author": "Laurent Mazare",
+    "authorEmail": "lmazare@gmail.com",
+    "framework": "PyTorch v1.0.0.dev20181116",
+    "codeURL": "https://github.com/LaurentMazare/deep-models/tree/master/cifar-10",
```
Comment (codyaustun) [inline CIFAR10/inference/laurentmazare_resnet9_1p100_pytorch.json]: Can you add the commit hash either to the `codeURL` or as a separate `commitHash` field?

**Review by codyaustun — CHANGES_REQUESTED** [review]
Thanks for the submission! It looks good overall. Please include the commit hash for your code.

File: CIFAR10/inference/laurentmazare_resnet9_1p100_pytorch.json (hunk 1 above)
Comment (LaurentMazare) [reply CIFAR10/inference/laurentmazare_resnet9_1p100_pytorch.json]: Sure, I just pushed a change adding the commit hash to codeURL.

**Review by codyaustun — APPROVED** [review]
